src/RevisionPopupWidget.js

```diff
@@ -162,7 +162,11 @@ RevisionPopupWidget.prototype.updateData = function ( data = {} ) {
 	// Comment content
 	this.commentLabel.setLabel(
 		new OO.ui.HtmlSnippet(
-			Tools.bidiIsolate( $.parseHTML( data.comment ) )
+			data.comment === undefined ?
+				$( '<span>' )
+					.addClass( 'history-deleted' )
+					.text( mw.msg( 'whowrotethat-revision-edit-summary-hidden' ) ) :
```

Comment:
Can the MediaWiki message `revdelete-summary-hid` be used here instead? To reduce duplication and translation efforts.

Reply:
I noticed comment https://github.com/wikimedia/WhoWroteThat/blob/c58d94ca7e54a50c21bea62e50822f5c5a518869/src/RevisionPopupWidget.js#L58. Would using `revdelete-summary-hid` create  the same concern regarding using a native Wikimedia message that might include parser functions?

Reply:
I noticed comment: 
https://github.com/wikimedia/WhoWroteThat/blob/c58d94ca7e54a50c21bea62e50822f5c5a518869/src/RevisionPopupWidget.js#L58
Would using `revdelete-summary-hid` create the same concern regarding using a native MediaWiki message that might include parser functions? If so, should I also add that comment here?

Reply:
Ah, that makes sense. In that case, should we add `{{msg-mw|revdelete-summary-hid}}` to the qqq docs to highlight the fact that it's the same message?

Reply:
I added the comment to qqq docs and also to the code just in case someone is only looking at the code.